include/compat.h

```diff
@@ -1,3 +1,24 @@
+/*! \page License
+   KQ is Copyright (C) 2002 by Josh Bolduc
+
+   This file is part of KQ... a freeware RPG.
+
+   KQ is free software; you can redistribute it and/or modify
+   it under the terms of the GNU General Public License as published
+   by the Free Software Foundation; either version 2, or (at your
+   option) any later version.
+
+   KQ is distributed in the hope that it will be useful,
+   but WITHOUT ANY WARRANTY; without even the implied warranty of
+   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+   GNU General Public License for more details.
+
+   You should have received a copy of the GNU General Public License
+   along with KQ; see the file COPYING.  If not, write to
+   the Free Software Foundation,
+       675 Mass Ave, Cambridge, MA 02139, USA.
+*/
+
 #pragma once
```

Comment:
Would be good to eliminate this file altogether - it was supposed to be some quick definitions to paper over the cracks between Allegro 4 and SDL2

Reply:
I agree. I scripted out something to first remove the header from all files, then re-insert a standard one into all of them.

Although, I would probably prefer to just remove it from all files and have a LICENSE.md or something in the project root. Then there only ever needs to be 1 change (like updating the Copyright year).

I think we also planned to remove the outdated source files outside of the `src/` and `include/` directories, as those were very Allegro- and C-specific.

Reply:
I'd leave them, but just remove the \\page directive from all but one (on kq.cpp.) I thought for a moment we didn't actually include the licence anywhere but (as the header says!) it's in the file COPYING. I don't know what we do about our dependency libraries - it is probably enough to reference them at the end of the README file (just say KQ depends on XXX library copyright YYY and give the project URL)